packages/location-field/package.json

```diff
@@ -14,7 +14,13 @@
     "@opentripplanner/geocoder": "^1.1.2",
     "@opentripplanner/humanize-distance": "^1.1.0",
     "@opentripplanner/location-icon": "^1.4.0",
+    "@opentripplanner/core-utils": "^4.1.0",
+    "@opentripplanner/geocoder": "^1.1.0",
+    "@opentripplanner/humanize-distance": "^1.1.0",
+    "@opentripplanner/location-icon": "^1.3.0",
```

Comment:
Why were some of these packages downgraded?

Reply:
I had run a lerna command that I really didn't understand what it was supposed to do and i didn't catch that it added these duplicate refs with lower version numbers. Fixed now.